hudi-cli/src/test/java/org/apache/hudi/cli/commands/TestFileSystemViewCommand.java

```diff
@@ -0,0 +1,245 @@
+
+    metaClient = HoodieCLI.getTableMetaClient();
+
+    partitionPath = HoodieTestCommitMetadataGenerator.DEFAULT_FIRST_PARTITION_PATH;
+    String fullPartitionPath = tablePath + "/" + partitionPath;
+    new File(fullPartitionPath).mkdirs();
+
+    // Generate 2 commits
+    String commitTime1 = "1";
+    String commitTime2 = "2";
+
+    String fileId1 = UUID.randomUUID().toString();
+
+    // Write date files and log file
+    new File(fullPartitionPath + "/" + FSUtils.makeDataFileName(commitTime1, TEST_WRITE_TOKEN, fileId1))
+        .createNewFile();
+    new File(fullPartitionPath + "/"
+        + FSUtils.makeLogFileName(fileId1, HoodieLogFile.DELTA_EXTENSION, commitTime1, 0, TEST_WRITE_TOKEN))
+        .createNewFile();
+    new File(fullPartitionPath + "/" + FSUtils.makeDataFileName(commitTime2, TEST_WRITE_TOKEN, fileId1))
+        .createNewFile();
+    new File(fullPartitionPath + "/"
+        + FSUtils.makeLogFileName(fileId1, HoodieLogFile.DELTA_EXTENSION, commitTime2, 0, TEST_WRITE_TOKEN))
+        .createNewFile();
+
+    // Write commit files
+    new File(tablePath + "/.hoodie/" + commitTime1 + ".commit").createNewFile();
+    new File(tablePath + "/.hoodie/" + commitTime2 + ".commit").createNewFile();
+
+    // Reload meta client and create fsView
+    metaClient = HoodieTableMetaClient.reload(metaClient);
+
+    fsView = new HoodieTableFileSystemView(metaClient, metaClient.getActiveTimeline(), true);
+  }
+
+  /**
+   * Test case for 'show fsview all'.
+   */
+  @Test
+  public void testShowCommits() {
+    // Test default show fsview all
+    CommandResult cr = getShell().executeCommand("show fsview all");
+    assertTrue(cr.isSuccess());
+
+    // Get all file groups
+    Stream<HoodieFileGroup> fileGroups = fsView.getAllFileGroups(partitionPath);
+
+    List<Comparable[]> rows = new ArrayList<>();
+    fileGroups.forEach(fg -> fg.getAllFileSlices().forEach(fs -> {
+      int idx = 0;
+      // For base file only Views, do not display any delta-file related columns
+      Comparable[] row = new Comparable[8];
+      row[idx++] = fg.getPartitionPath();
+      row[idx++] = fg.getFileGroupId().getFileId();
+      row[idx++] = fs.getBaseInstantTime();
+      row[idx++] = fs.getBaseFile().isPresent() ? fs.getBaseFile().get().getPath() : "";
+      row[idx++] = fs.getBaseFile().isPresent() ? fs.getBaseFile().get().getFileSize() : -1;
+      row[idx++] = fs.getLogFiles().count();
+      row[idx++] = fs.getLogFiles().mapToLong(HoodieLogFile::getFileSize).sum();
+      row[idx++] = fs.getLogFiles().collect(Collectors.toList()).toString();
+      rows.add(row);
+    }));
+
+    Function<Object, String> converterFunction =
+        entry -> NumericUtils.humanReadableByteCount((Double.parseDouble(entry.toString())));
+    Map<String, Function<Object, String>> fieldNameToConverterMap = new HashMap<>();
+    fieldNameToConverterMap.put("Total Delta File Size", converterFunction);
```

Comment:
These string literals are the same as `FileSystemViewCommand`. We can define them as constant fields and reuse them here?